src/runtime-rs/Makefile

```diff
@@ -21,8 +21,18 @@ ARCH_DIR = arch
 ARCH_FILE_SUFFIX = -options.mk
 ARCH_FILE = $(ARCH_DIR)/$(ARCH)$(ARCH_FILE_SUFFIX)
 
+
+ifeq ($(ARCH), s390x)
+##TARGET default: build code
+default:
+	@echo "s390x not support currently"
+	exit 0
+else
+default: runtime show-header
+endif
+
 ifeq (,$(realpath $(ARCH_FILE)))
-    $(error "ERROR: invalid architecture: '$(ARCH)'")
+	$(error "ERROR: invalid architecture: '$(ARCH)'")
```

Comment:
Please keep the original indent chars as it was (tabs or spaces).

Reply:
fixed